roles/v2.4/kiali-deploy/defaults/main.yml

```diff
@@ -181,7 +181,6 @@ kiali_defaults:
       istio_identity_domain: "svc.cluster.local"
       istio_injection_annotation: "sidecar.istio.io/inject"
       istio_sidecar_annotation: "sidecar.istio.io/status"
-      istio_sidecar_injector_config_map_name: "istio-sidecar-injector"
```

Comment:
we need to delete this from the "default" role, too, correct?

https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/defaults/main.yml#L185